src/libraries/System.IO.Compression.ZipFile/src/System.IO.Compression.ZipFile.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <TargetFrameworks>$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Browser;$(NetCoreAppCurrent)</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Browser;$(NetCoreAppCurrent)</TargetFrameworks>
```

Comment:
@eerhardt as you added the Browser and Unix configurations in https://github.com/dotnet/runtime/commit/5b0c6dd5756eea63a57b1f69b182f7386da0f631, do you remember why you needed a separate Browser build?

Was it just to satisfy the build graph (because at that time there was no rid-less default implementation) or because you actually compile differently for Browser?

If the former, then we want to remove the Browser tfm.

Reply:
@ViktorHofer done